ember-cli-build.js

```diff
@@ -112,7 +113,7 @@ module.exports = function (defaults) {
                 plugins: postcssPlugins()
             }
         },
-        fingerprint: disabled,
+        fingerprint: {enabled: true},
```

Comment:
Fingerprinting is only enabled in production by default, enabling it for development too should hopefully help catch those edge cases where we expect a non-fingerprinted asset